dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TenantServiceImpl.java

```diff
@@ -95,7 +95,8 @@
             return result;
         }
 
-        if (checkTenantExists(tenantCode)) {
+        // check if tenant exists in database or named "default"
+        if (checkTenantExists(tenantCode) || "default".equalsIgnoreCase(tenantCode)) {
```

Comment:
Why not allow tenants to be set to 'default' ?

Reply:
![图片](https://user-images.githubusercontent.com/23203149/133557913-946ed17e-e519-4a5f-8dae-9e9f0525ddc9.png)

There's already a tenant named "default" which is a default tenant if no other tenant could be chosen, if we create a new tenant named "default", it hard to figure out which one is the "real" tenant we need.

Reply:
I think the front-end display should be repaired here. When there is already a tenant, the virtual tenant should not be displayed

Reply:
Yes, you are right.
I think the front-end should display noting if no tenant was created.
if the user want to create a workflow when no tenant exists, he should create a new tenant first.

Reply:
I have different suggestion. if the user want to create a workflow when no tenant exists, in order not to affect workflow creation the tenant id will be -1, and now the tenant code is 'default'. I think that's how default appears.

Reply:
So the question is how to avoid 'default' tenant in front-end and user created tenant named 'default' both appears.
Should we add a mark in the front-end to distinguish? or just remove the 'default' option when tenant list is not empty.

Reply:
I mean, if the tenant exists, it will be displayed, and the default of this enumeration will not be displayed. If it does not exist, the default of enumeration will be displayed